classes/controller/AdminController.php

```diff
@@ -297,6 +297,9 @@ class AdminControllerCore extends Controller
 
 	protected $logged_on_addons = false;
 
+	/** @var bool if logged employee has access to AdminImport */
+	protected $canImport = false;
```

Comment:
Hi,

In PrestaShop variable must be in snakeCase, please check http://doc.prestashop.com/display/PS16/Coding+Standards

Best regards.